WPFHospitalEditor/Building1.xaml.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
```

Comment:
Building1 and Building2 must not be in separate classes. Make one class Building where you define by constructor which building data you should use. This is temporary solution!